tests/unit/AccountManagerLinkRendererTest.tsx

```diff
@@ -0,0 +1,146 @@
+import {fireEvent, render, screen} from '@testing-library/react-native';
+import React from 'react';
+import Onyx from 'react-native-onyx';
+import AccountManagerLinkRenderer from '@components/HTMLEngineProvider/HTMLRenderers/AccountManagerLinkRenderer';
+import Navigation from '@libs/Navigation/Navigation';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import waitForBatchedUpdatesWithAct from '../utils/waitForBatchedUpdatesWithAct';
+
+jest.mock('@libs/Navigation/Navigation', () => ({
+    navigate: jest.fn(),
+}));
+
+jest.mock('@components/HTMLEngineProvider/htmlEngineUtils', () => ({
+    isChildOfRBR: jest.fn(() => false),
+    getFontSizeOfRBRChild: jest.fn(() => 15),
+}));
+
+jest.mock('react-native-render-html', () => {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    const actual = jest.requireActual('react-native-render-html');
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+    return {
+        ...actual,
+        TNodeChildrenRenderer: ({tnode}: {tnode: {mockText?: string}}) => tnode.mockText ?? 'Account Manager',
+    };
+});
+
+const ACCOUNT_MANAGER_REPORT_ID = '123456789';
+
+const createMockTNode = (text = 'Account Manager') => ({
+    type: 'phrasing',
+    mockText: text,
+    domNode: {
+        name: 'account-manager-link',
+        attribs: {},
+    },
+    children: [],
+    parent: null,
+    nodeIndex: 0,
+});
+
+describe('AccountManagerLinkRenderer', () => {
+    beforeAll(() =>
+        Onyx.init({
+            keys: ONYXKEYS,
+        }),
+    );
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+        return Onyx.clear();
+    });
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should render the link with correct text', async () => {
+        await Onyx.merge(ONYXKEYS.ACCOUNT_MANAGER_REPORT_ID, ACCOUNT_MANAGER_REPORT_ID);
+        await waitForBatchedUpdatesWithAct();
```

Comment:
No need to merge the onyx data for this test case.

Also, no need to use `waitForBatchedUpdatesWithAct` after `await Onyx.merge` for all cases.